api/types/types.go

```diff
@@ -556,6 +556,21 @@ type DiskUsageOptions struct {
 	Types []DiskUsageObject
 }
 
+// ContainerUsage contains container usage part of response of Engine API:
+// GET "/system/df"
+type ContainerUsage struct {
+	ID         string `json:"Id"`
+	Image      string
+	ImageID    string
+	Command    string
+	Created    int64
+	Names      []string
+	SizeRw     int64 `json:",omitempty"`
+	SizeRootFs int64 `json:",omitempty"`
+	Status     string
```

Comment:
(Just a random comment) the `Status` field is kinda horrible, as it's pre-formatting the output (e.g. `"Up 4 seconds"`). We don't have a good solution for that yet though (perhaps we should add a `ServerTime` header to API responses, so that the `container.State` field can be returned, and relative times can be handled client side. I know there's some bugs in that area in the CLI currently (wrong duration shown because it compares "local" time with "server-based" times)